pkg/experimental/rpm.bzl

```diff
@@ -304,6 +327,19 @@ install -d %{{buildroot}}/{0}
                 install_script_pieces.append(install_dir_stanza_fmt.format(
                     d,
                 ))
+        if PackageSymlinkInfo in elem:
```

Comment:
Why don't we do the duplicate checks here rather than above?
It would only be one instance of the loop to get the destination elements.
It also seems like the error would be more useful if the fully formed destination was in the error.

Reply:
The initial reason was to avoid having too much clutter in the loops.

You do make good points, though.  I'd rather not look into this now, but I've tracked it as a part of #191.